lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -464,6 +468,9 @@ <h3 class="error-msgs" id="error-msg"></h3>
      $('#course_price_link').click(function () {
        reset_input_fields();
      });
+     $('#query_registration_code_link').click(function () {
```

Comment:
Can you create a debt ticket, and leave a `TODO` linking to the ticket? This needs to be cleaned up.